static/templates/edit_bot.hbs

```diff
@@ -1,36 +1,45 @@
+<div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+    <h3 class="inline-block" id="change_full_name_modal_label">{{t "Edit bot" }}</h3>
+    <div class="alert-notification change_full_name_info"></div>
+</div>
 <form class="edit_bot_form form-horizontal" data-target="edit-bot" data-title="{{t 'Edit bot' }}" data-user-id="{{bot.user_id}}"
   data-email="{{bot.email}}" data-type="{{bot.bot_type}}">
     <div class="bot_edit_errors alert alert-error hide"></div>
-    <div class="">
-        <label>{{t "Bot email" }}</label>
-        <div class="edit_bot_email">{{bot.email}}</div>
-    </div>
-    <div class="edit-bot-form-box">
+    <div class="edit_bot_form_container">
         <div class="">
-            <label for="edit_bot_name">{{t "Full name" }}</label>
-            <input id="edit_bot_name" type="text" name="bot_name" class="edit_bot_name required" value="{{bot.full_name}}" maxlength=50 />
-            <div><label for="edit_bot_name" generated="true" class="text-error"></label></div>
+            <label>{{t "Bot email" }}</label>
+            <div class="edit_bot_email">{{bot.email}}</div>
         </div>
-        <div class="edit-bot-owner">
-            <label for="bot_owner_select">{{t "Owner" }}</label>
-            <div class="select-form">
-                {{> bot_owner_select }}
+        <div class="edit-bot-form-box">
+            <div class="">
+                <label for="edit_bot_name">{{t "Full name" }}</label>
+                <input id="edit_bot_name" type="text" name="bot_name" class="edit_bot_name required" value="{{bot.full_name}}" maxlength=50 />
+                <div><label for="edit_bot_name" generated="true" class="text-error"></label></div>
+            </div>
+            <div class="edit-bot-owner">
```

Comment:
This dropdown menu's color seems a bit off, i.e., it is the same color as the background instead of being white.

![Screenshot_2](https://user-images.githubusercontent.com/25329595/77558529-0c394c00-6ee1-11ea-87b2-77f81ea16a04.png)

Reply:
ok

Reply:
actually I don't use any new CSS I had used already available CSS

Reply:
You did change/add a number of classes and ids. That probably affected the design of these elements.